### PR TITLE
Change `Symbol#to_bool` to be case-insensitive

### PR DESCRIPTION
**Pull request by yujideveloper**
I changed `Symbol#to_bool` to be case-insensitive (e.g. `:TRUE` is true)
And  I included some cosmetic changes.

### CODE REVIEW

File: lib/to_bool.rb [hunk 1]
```diff
@@ -20,21 +20,21 @@ def to_bool
...
   end
 
-  alias_method :to_boolean, :to_bool
+  alias to_boolean to_bool
 end
 
 class Symbol
   def to_bool
-    self == :true
+    self.downcase == :true
```
Comment (bricker) [inline lib/to_bool.rb:36]: Can you add a test case for `:TRUE` please?

Comment (yujideveloper) [reply lib/to_bool.rb:36]: Hi, @bricker 

Thanks for your reviewing.
I added a test case for `:TRUE` and some missing test cases for `String#to_bool`.

Comment (bricker) [reply lib/to_bool.rb:36]: Thanks!

**Review by bricker — APPROVED** [review]
Thanks!

**Comment by bricker** [comment]
This has been released with v1.2.0, thank you!

**Pull request merged**